formatter_test.go

```diff
@@ -259,3 +260,46 @@ func TestCycle(t *testing.T) {
 	*iv = *i
 	t.Logf("Example long interface cycle:\n%# v", Formatter(i))
 }
+
+func Test(t *testing.T) {
+	date := time.Date(2006, 1, 2, 3, 4, 5, .678901e9, time.Local)
+
+	s := fmt.Sprintf("%s", Formatter(date))
+
+	if date.String() != s {
+		t.Errorf("expected %s, got %s", date.String(), s)
+	}
+}
+
+type StringerStruct struct{}
+
+func (t StringerStruct) String() string {
+	return "StringerStruct"
+}
+
+func TestStringer(t *testing.T) {
+	s := new(StringerStruct)
+
+	str := fmt.Sprintf("%s", Formatter(s))
```

Comment:
Formatter ignores format verbs other than v with # and ' ' (space) flags. This format string needs to use `% #v` instead of `%s`. Currently, it's not testing what it's meant to test.